### DISCUSSION TITLE
Someone knows why I can't update from web interface, only at command line?

### DISCUSSION THREAD

**Discussion by hysonlee** [datenstrom/community#265] [Ask a question] [answered]
hi all,
Just try to update from webinterface , an Server error reported as: 
`Something went wrong. Can't update file 'system/plugins/blog.php'!`

then I try to update via ssh console, it worked. I don't know why.
```

root@lmde8:/home/wwwroot/yellow.hyson.tk# php yellow.php version
Datenstrom Yellow 0.7.1, PHP 7.0.19, CLI Linux
YellowBlog 0.7.2 - Update available
YellowCommand 0.7.2
YellowCore 0.7.1
YellowEdit 0.7.2
YellowImage 0.7.2
YellowLanguage 0.7.2
YellowMarkdown 0.6.5
YellowUpdate 0.7.3
YellowThemeFlatsite 0.6.13
root@lmde8:/home/wwwroot/yellow.hyson.tk# php yellow.php update
YellowBlog 0.7.2
Yellow update: Website updated, 1 update installed
```

**Comment by schulle4u** [comment, marked as answer]
You should not update yellow from the commannd line as root, this will break file ownership and you'll have to chown all updated files to www-data or the appropriate website user afterwards. So I'm assuming a permission problem when using the webinterface updater.
